.woodpecker.yml

```diff
@@ -1,19 +1,23 @@
 variables:
-  - &rust_image "rustlang/rust:nightly-slim"
+  - &rust_image "rustlang/rust:nightly"
+  - &install_binstall "wget https://github.com/cargo-bins/cargo-binstall/releases/latest/download/cargo-binstall-x86_64-unknown-linux-musl.tgz && tar -xvf cargo-binstall-x86_64-unknown-linux-musl.tgz && cp cargo-binstall /usr/local/cargo/bin"
```

Comment:
I've combined these up here now.

Reply:
Nice! Is there a way to use the same downloaded b-install across the different steps instead of downloading it repeatedly? Downloading it more than once seems like it will add unnecessary time to the build.

Reply:
Not that I know of, because I'm not exactly sure how to properly do resource sharing in woodpecker. But they all finish within 15s anyway, and its done in parallel: https://woodpecker.join-lemmy.org/repos/140/pipeline/303/3